openvino-requirements.txt

```diff
@@ -0,0 +1,2 @@
+openvino==2021.4
+openvino-dev==2021.4
```

Comment:
Could you, please, explain why we want to use the pip package instead of the OpenVINO distributive?
And why do we need two packages, openvino and openvino-dev? Isn't it enough one of them?

Reply:
openvino-dev contains a collection of tools (AC, MO, POT, etc.), while openvino contains IE.
Usage of pypi packages is required from the SC/OTE CLI side. Backward compatibility with the classic distribution is provided.